blockchain/chain_sync/src/tipset_syncer.rs

```diff
@@ -1163,16 +1163,16 @@ async fn validate_tipset<DB: Blockstore + Store + Clone + Send + Sync + 'static,
     debug!("Tipset keys: {:?}", full_tipset_key.cids);
 
     for b in blocks {
-        let validation_fn = tokio::task::spawn(validate_block::<_, C>(
+        let validation_fn = tokio::task::spawn(validate_block(
```

Comment:
Is there a reason you're spawning things twice?

```suggestion
        let validation_fn = validate_block(
```